core-feature-pack/src/main/resources/content/bin/domain.sh

```diff
@@ -131,8 +135,6 @@ if [ "x$SERVER_SET" = "x" ]; then
             JVM_OPTVERSION="-server $JVM_OPTVERSION"
         fi
     fi
-else
```

Comment:
Why this else block needs to be removed?

Anyway, [original jira](https://issues.jboss.org/browse/WFCORE-3825) is just about windows scripts, if you want to change domain.sh script, please describe it in [original jira](https://issues.jboss.org/browse/WFCORE-3825) description and PR description or create new jira and PR for that.